_docs/_latest/configure/scanner/default-rules.md

```diff
@@ -40,6 +40,9 @@ resources.
   * Forbid any Cloud IAP bypasses on all resources in my organization, when Cloud IAP is enabled.
   * Allow direct access from debug IPs and internal monitoring hosts.
 
+## External Project Access
+  * Find any users in your org that may have access to projects outside of your org.
```

Comment:
Since folder is support, we should change this to:
```
 * Find any users in your org that may have access to projects outside of your allowed org or folder.
```